server/fishtest/templates/elo_results.mak

```diff
@@ -2,14 +2,14 @@
 
 <%!
```

Comment:
Missing import:
```python
  from fishtest.util import is_active_sprt_ltc
```

Reply:
Honestly it should just be a method of `run` lol